src/controls/QskListView.cpp

```diff
@@ -151,6 +151,29 @@ QskColorFilter QskListView::graphicFilterAt( int row, int col ) const
     return QskColorFilter();
 }
 
+QskAspect::Subcontrol QskListView::rowSubControl( int row ) const noexcept
+{
+    if ( alternatingRowColors() )
+    {
+        if ( row == selectedRow() )
+        {
+            return Cell;
+        }
+        return row % 2 == 0 ? Cell : QskAspect::NoSubcontrol;
```

Comment:
@uwerat this is somewhat for keeping compatible, since in the previous implementation alternating color ment colored box node, no box node, colored box node, ...